commons/passive/garmin/garmin_generic_heartrate.avsc

```diff
@@ -0,0 +1,46 @@
+{
+   "name":"GarminGenericHeartRate",
+   "type":"record",
+   "namespace":"org.radarcns.passive.garmin",
+   "doc":"Provides the data about real time Heart rate data.",
+   "fields":[
+
+      {
+            "name":"time",
+            "type":"double",
+            "doc":"Last updated time."
+         },
+         {
+            "name":"timeReceived",
+            "type":"double",
+            "doc":"Device receiver timestamp in UTC (s)."
+         },
+
+      {
+         "name":"currentHeartRate",
+         "type":"int",
+         "doc":"Current heart rate (bpm)."
+      },
+      {
+         "name":"currentRestingHeartRate",
+         "type":"int",
+         "doc":"Current resting heart rate (bpm)."
+      },
+      {
+         "name":"dailyHighHeartRate",
+         "type":"int",
+         "doc":"Daily high heart rate (bpm)."
+      },
+      {
+         "name":"dailyLowHeartRate",
+         "type":"int",
+         "doc":"Daily low heart rate (bpm)."
+      },
+      {
+         "name":"heartRateSource",
+         "type":"string",
+         "doc":"Heart rate source. Possible values are NO_SOURCE, HR_STRAP, OHR_NO_LOCK, OHR_LOCKED."
```

Comment:
is there some documentation about what those values mean?

Reply:
Meaning of these values are not available in any current documentation. We will reach out to Garmin to understand this.

Reply:
If this set of values is fixed, please define it as an enum, with first value `UNKNOWN` and also the default value `UNKNOWN`. See https://github.com/RADAR-base/RADAR-Schemas/blob/def38542256c4e48ebeaf479c2542000c649ff6d/commons/passive/phone/phone_usage_event.avsc#L12.

Reply:
Converted to enum in latest checkin.